lib/node_modules/@stdlib/math/base/special/riemann-zeta/lib/main.js

```diff
@@ -265,7 +265,7 @@ function zeta( s ) {
 			tmp = sinpi( 0.5*sc ) * 2.0 * zeta( s );
 			r = gammaln( s );
 			r -= s * ln( TWO_PI );
-			if ( r > MAX_LN ) {
+			if ( r > floor( FLOAT64_MAX_LN ) ) {
```

Comment:
@gunjjoshi Rather than dynamically compute, make L66

```javascript
var MAX_LN = floor( FLOAT64_MAX_LN );
```

In order to avoid unnecessary repeated computation.